lib/kube/utils/utils.go

```diff
@@ -196,3 +197,11 @@ func CheckOrSetKubeCluster(ctx context.Context, p KubeServicesPresence, kubeClus
 	}
 	return kubeClusterNames[0], nil
 }
+
+type WriterCloserWrapper struct {
```

Comment:
You can delete this - we already have `utils.NopWriteCloser` which does exactly this.